modules/localStorage.js

```diff
@@ -0,0 +1,7 @@
+// add data to local storage
+
+function addToLocalStorage(data) {
```

Comment:
- [ ] Please, consider using arrow function syntax, as required in the project (use ES6 syntax). For example, here you can go on with 
```
const addToLocalStorage = (data) => {
....
```

- [ ] Also, will be better to create an additional module for storing data, then import it here in the `index.js` file